packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseTrendingProjectsItem.tsx

```diff
@@ -0,0 +1,131 @@
+import { Box, Typography, SxProps } from '@mui/material'
+import React from 'react'
+import { calculateFriendlyFileSize } from '../../utils/functions/calculateFriendlyFileSize'
+import { useGetEntityHeader } from '../../synapse-queries'
+import { NavigateNext } from '@mui/icons-material'
+import { BackendDestinationEnum, getEndpoint } from '../../utils/functions'
+import { ReactComponent as EgressIcon } from '../../assets/homepage/egress.svg'
+
+import { ReactComponent as UsersIcon } from '../../assets/homepage/users.svg'
+export type SynapseTrendingProjectItemProps = {
+  rank: number
+  rowValues: (string | null)[]
+  entityIdColIndex: number
+  nUniqueUsersColIndex: number
+  egressSizeGbColIndex: number
+  isMobileView: boolean
+}
+
+const formatter = Intl.NumberFormat('en')
+export const trendingProjectsGridTemplateColumns: SxProps = {
+  gridTemplateColumns: {
+    xs: '100px auto 40px',
+    sm: '100px auto 170px 150px 40px',
+  },
+}
+
+export const SynapseTrendingProjectItem: React.FunctionComponent<
+  SynapseTrendingProjectItemProps
+> = ({
+  rank,
+  rowValues,
+  entityIdColIndex,
+  nUniqueUsersColIndex,
+  egressSizeGbColIndex,
+  isMobileView,
+}) => {
+  const entityId = rowValues[entityIdColIndex]
+  const egressSize = rowValues[egressSizeGbColIndex]
+  const { data: entityHeader } = useGetEntityHeader(entityId!, undefined, {
+    enabled: !!entityId,
+  })
+  const friendlyEgressSize = egressSize
+    ? calculateFriendlyFileSize(parseFloat(egressSize) * 1e9)
+    : ''
+
+  const userCount = rowValues[nUniqueUsersColIndex]
+  const friendlyUserCount = userCount
+    ? formatter.format(parseInt(userCount))
+    : ''
+  let backgroundColor = '#223549'
```

Comment:
```suggestion
  let backgroundColor = theme.palette.primary[700]
```

Reply:
Some type issues here so this specific change caused an error, but I worked it out